README.md

```diff
@@ -148,3 +150,17 @@ root@pyth-dev# usermod -u 1002 -g 1004 -s /bin/bash pyth
 ```
 
 Finally, in docker extension inside VS Code click right and choose "Attach VS Code". If you're using a remote host in VS Code make sure to let this connection be open.
+
+## Deployment
+
+Oracle program upgrades are managed by the Pythian Council multisig. The steps are:
+
+1. Create a release branch from `main`.
+2. Include binaries for both the Solana mainnet and Pythnet oracle programs (`pyth_oracle_solana.so` and `pyth_oracle_pythnet.so`).
```

Comment:
This is not really 2 it's more like : 
`1. Create a release branch from `main`. The release will contain binaries for ...`